manifest.yml

```diff
@@ -0,0 +1,21 @@
+---
+memory: 256MB
```

Comment:
memory is below.  I think you want to delete this one and make the other one 256